userstories.excalidraw

```diff
@@ -0,0 +1,2555 @@
+{
```

Comment:
Is this file intented to be here?

Reply:
actually not. i created my userstories with the excalidraw extension @VS Code